commands/station.js

```diff
@@ -82,6 +83,7 @@ export const station = async ({ json, experimental }) => {
 
   const modules = [
     zinniaRuntime.run({
+      STATION_ID: (await getStationId()).publicKey,
```

Comment:
If calling keytar is going to trigger a UI prompt on macOS, I strongly prefer to make this call only once during station-core startup.

I am also proposing to include station id in the `ping` telemetry we are submitting here, which is yet another argument for moving the `getStationId()` call somewhere else.

https://github.com/filecoin-station/core/blob/f8fd023d90715202ef5d23d253776b8036ee6456/lib/telemetry.js#L52-L56

Reply:
Hmm, I think I got confused. This file is called only once, and then it's `zinniaRuntime.run` that handles module updates and (re) starts the runtime.

In that light, it's okay to call `getStationId()` inside the `station()` function.

However, I would find the code cleaner if this call were happening somewhere closer to the top, where we are also reading FIL_WALLET_ADDRESS, and the ID was stored in a variable.

```js
const { publicKey: STATION_ID } = await getStationId()
// ...
  const modules = [
    zinniaRuntime.run({
      STATION_ID,
      // ...
```